src/framework/filterControl/forms/MultipleSelectFilterForm.spec.jsx

```diff
@@ -76,12 +81,37 @@ describe('MultipleSelectFilterForm', () => {
 
         options.forEach((option) => {
           expect(testCase.find(
-            `.filterForm--multiSelect__checkbox:contains(${option})`).length
-          ).toBe(1);
+            `.filterForm--multiSelect__checkbox:contains(${option.value})`
+          ).length).toBe(1);
         });
       });
     });
 
+    it('options has different value and label ', () => {
```

Comment:
I would not have an `it` block, but rather define value and label to be different by default (in lines 11-15). We can have a small test for `OneOfOption` that checks label when only value is provided.